central/dbc/compile.go

```diff
@@ -0,0 +1,209 @@
+// The following code was copied from https://github.com/einride/can-go
```

Comment:
add apache2 license header in any new files

Reply:
added license header